packages/theme-chalk/src/carousel-item.scss

```diff
@@ -32,9 +32,14 @@
         }
       }
       &.is-active {
-        z-index: calc(getCssVar('index', 'normal') + 1);
+        z-index: calc(getCssVar('index', 'normal') + 10);
```

Comment:
This is partly added because the zIndex of the carousel-item in the previous version of the code may be incorrect. Now this part of the code is no longer needed. I will modify it again.